doc/classes/Control.xml

```diff
@@ -865,7 +865,7 @@
 			Tells the parent [Container] nodes how they should resize and place the node on the X axis. Use one of the [enum SizeFlags] constants to change the flags. See the constants to learn what each does.
 		</member>
 		<member name="size_flags_stretch_ratio" type="float" setter="set_stretch_ratio" getter="get_stretch_ratio" default="1.0">
-			If the node and at least one of its neighbours uses the [constant SIZE_EXPAND] size flag, the parent [Container] will let it take more or less space depending on this property. If this node has a stretch ratio of 2 and its neighbour a ratio of 1, this node will take two thirds of the available space.
+			If the node and at least one of its neighbours uses the [constant SIZE_EXPAND] size flag, the parent [Container] will let it take more or less space depending on this property. If this node has a stretch ratio of 2 and its neighbour a ratio of 1, this node will take two thirds of the available space. If using the editor slider with the mouse, hold the Control key to round values to the nearest integer, or the Shift key to slow down the value rate of change.
```

Comment:
Pretty sure this should not be in the class documentation.

It might still be nice to have it in a tooltip or in some editor documentation.

Reply:
I agree... I just put it there because I wanted it documented, but I was quite sure it was not the right place. I'll see if I can put it into a tool tip or something... Didn't look for that last night when I did it.

Reply:
See update... removed doco and added tooltip.